ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth63.java

```diff
@@ -51,6 +51,7 @@
 
     private long connectedTime = System.currentTimeMillis();
     private long processingTime = 0;
+    private long lastReqSentTime;
```

Comment:
This is from reverted commit (about more than one hash checks).
I left it as next task will be to bring this fix back.